packages/salesforcedx-vscode-core/src/DeveloperLogDebugLevels.ts

```diff
@@ -0,0 +1,49 @@
+export class DeveloperLogDebugLevels {
```

Comment:
I added this since both the turn on and off command had to work with the state of the developer_log debug levels. Not sure if this is overkill or not so would like feedback on this.

Reply:
I like this.